shared/src/persistence/s3/isFileExists.test.ts

```diff
@@ -48,6 +47,7 @@ describe('isFileExists', () => {
       await isFileExists({
         applicationContext,
         key: 'jenny-not-found',
+        useTempBucket: false,
```

Comment:
Similar question to above, just for my learning 😄 If `useTempBucket` is marked as optional and defaulted to false, why does it have to be added here?

Reply:
correct, I can remove those from this file since they defaulted to false in the function.